### ISSUE TITLE
Window shadow inconsistency

### ISSUE THREAD

**Issue by marmis85** [numixproject/numix-gtk-theme#270]
Some GTK applications (like nautilus, gedit and most gnome applications) have a nice and smooth shadow but other applications (like gnome-terminal, empathy but also gimp and chrome when not maximized) have a thick and pointy shadow

![screenshot](https://cloud.githubusercontent.com/assets/1887387/11256050/0b320210-8e4a-11e5-9305-49adfdd4e62e.png)

I've noticed a pattern where GTK applications using the toolbar with CSDs have the nice shadow and applications with the old titlebar have the ugly shadow

I'm using GNOME and GTK 3.18 and Numix from git master on Arch Linux

**Comment by simongrund1** [comment]
I can confirm this issue. For me too, the difference appears to be betwen GTK2 vs. 3 applications, or those using vs. not using CSDs. To me it looks like the borders around GTK2/non-CSD windows are "too small" for the shadow.

In addition, I observed two things:

1) Restarting the GNOME session using Alt+F2, then running "r", makes the shadows look more consistent, albeit not the same. However, when maximizing and unmaximizing the window or otherwise resizing the window, its shadows revert to this inconsistent bahavior.
Similarly, opening a new window of a GTK2 application has "pretty" shadows initially, but maximizing/unmaximizing or resizing makes them inconsistent again.

2) The boxes around GTK2 applications seem to be smaller than for GTK3 applications, which is visible when trying to resize the windows (meaning that GTK3 applications have a larger area that can be used for resizing).

EDIT:
Running GNOME with GTK 3.18.6-1 and GTK 2.24.29-1 on Arch Linux with Numix from the official repository (numix-themes, 2.5.1-1).

**Comment by dimitriscsd** [comment]
I am having the same problem on Ubuntu with gnome shell 3.16.4

Those shadows look ugly :(

**Comment by hotice** [comment]
This also occurs in Ubuntu 16.04 (GNOME Shell 3.18.4, GTK 3.18.9), for instance with Dconf Editor, Rhythmbox, Atom (text editor) and so on.

**Comment by khurshid-alam** [comment]
Can't reproduce the issue on my machine. Is this happening under ubuntu-gnome or unity? Are you all using **fresh** install of Ubuntu 16.04 beta2? Upgrading libgtk-3.0 usually leaves old library files behind.

Also please use Numix from ppa.

![shadow](http://i.imgur.com/VTeAleW.png)

**Comment by hotice** [comment]
It occurs in GNOME Shell. It's fine in Unity. I'm not sure what you mean about old libraries, I didn't install beta 2 but an older version, but old libraries don't interfere, I even ran "sudo apt-get autoremove". And also, this bug does not occur with Adwaita... I just tried the latest Ubuntu GNOME 16.04 daily ISO (built yesterday) and the issue is present as expected: 

![image](https://cloud.githubusercontent.com/assets/325449/14522345/c4238996-0236-11e6-9f1b-175877b7ef2a.png)

**Comment by khurshid-alam** [comment]
@hotice Can you please check if this fixes the issue or not?

First install dependencies
`sudo apt-get install ruby-sass libgdk-pixbuf2.0-dev libxml2-utils libglib2.0-bin`

Install Numix from PPA. Go to `/usr/share/themes/Numix Daily` as super user.

Edit `gtk-3.0/scss/widgets/_window.scss` and add `box-shadow: 0 0 0 1px transparent;` under ssd section

``` scss
          &.ssd {
              box-shadow: 0 0 0 1px transparent;
              &.maximized { border-radius: 0; }
         }
```

Recompile the theme(run commands from theme's root directory or adjust your path):

``` sh
scss --update --sourcemap=none gtk-3.0/scss:gtk-3.0/dist
glib-compile-resources --sourcedir=gtk-3.0 gtk-3.0/gtk.gresource.xml
```

**Restart** gnome-shell.

**Comment by hotice** [comment]
@khurshid-alam I don't think you mean `gtk-3.0/scss/_widgets.scss` because there is no "ssd" in this file, as you can also see online @ https://github.com/shimmerproject/Numix/blob/master/gtk-3.0/scss/_widgets.scss

Edit: I found out what file you meant, it's `scss/widgets/_window.scss`

**Comment by khurshid-alam** [comment]
@hotice Yeah..it was a typo...Fixed.

**Comment by hotice** [comment]
@khurshid-alam I just tried it and it works, the shadow seems fixed!

**Comment by khurshid-alam** [comment]
This got fixed in 6485478945f58e90ad77a5e1ee060966a7bd7d2a
